content/docs/command-reference/init.md

```diff
@@ -14,106 +14,103 @@ DVC works best in a Git repository. This enables all features, providing the
 most value. For this reason, `dvc init` (without flags) expects to run in a Git
 repository root (a `.git/` directory should be present).
 
-The command [options](#options) can be used to start an alternative workflow for
-advanced scenarios:
-
-- [Initializing DVC in subdirectories](#initializing-dvc-in-subdirectories) -
-  support for monorepos, nested <abbr>DVC projects</abbr>, etc.
-- [Initializing DVC without Git](#initializing-dvc-without-git) - support for
-  SCM other than Git, deployment automation cases, etc.
-
 At DVC initialization, a new `.dvc/` directory is created for internal
 configuration and <abbr>cache</abbr>
 [files and directories](/doc/user-guide/dvc-files-and-directories#internal-directories-and-files),
 that are hidden from the user. This directory is automatically staged with
 `git add`, so it can be easily committed with Git.
 
+The command [options](#options) can be used to start an alternative workflow for
+advanced scenarios:
+
+- [Initializing DVC in subdirectories](#initializing-dvc-in-subdirectories)
+  (`--subdir`) - for monorepos and nested <abbr>DVC projects</abbr>
+- [Initializing DVC without Git](#initializing-dvc-without-git) (`--no-scm`) -
+  for very simple projects, SCM other than Git, deployment automation, among
+  other uses
+
+> Note that <abbr>DVC repositories</abbr> nested inside other DVC repos (for
```

Comment:
very hard to understand ... mostly because we use DVC repository term here probably.

Reply:
and probably because it comes almost at the very beginning - a bit unexpected

Reply:
Changed a little but I'm not sure how to make it super clear. It's just a minor note very specific for people looking for that specific use case (DVC + Git submodules). Don't you think they will get it?

Reply:
> because it comes almost at the very beginning

It's at the end of the main **Description** section. Should I move it to after all the other sub-headings? It would look as it pertains to the --no-scm explanation...

Reply:
it's fine let's keep it


> Don't you think they will get it?

I would expect from people who use submodules have the same intuition by default - isolated in Git, isolated in DVC - nothing special or specif. unless I'm missing something. That's why I'm a bit hesitant about all this changes and notes. It's hard to cover every case with a note.

Reply:
True, I can just remove it and handle them via support channel if needed. So far there was only one case that I noticed.